components/widgets/ui/src/main/resources/XWiki/DateTimePicker.xml

```diff
@@ -2890,7 +2890,7 @@ var SimpleDateFormat;
       <code>#template('colorThemeInit.vm')
 input.xwiki-date {
   background-image: url("$xwiki.getSkinFile('icons/silk/date.png')");
-  background-position: right center;
+  background-position: .3em center;
```

Comment:
Once you enter a date, the date is overlapping the icon and thus is unreadable. Also add a `2em` left padding.